src/libraries/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -505,6 +506,37 @@ public abstract partial class HttpHeaders : System.Collections.Generic.IEnumerab
         public bool TryAddWithoutValidation(string name, string? value) { throw null; }
         public bool TryGetValues(string name, [System.Diagnostics.CodeAnalysis.NotNullWhenAttribute(true)] out System.Collections.Generic.IEnumerable<string>? values) { throw null; }
     }
+    public readonly struct HttpHeadersNonValidated : System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, HeaderStringValues>>
+    {
+        public System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, HeaderStringValues>> GetEnumerator() { throw null; }
+        System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, HeaderStringValues>> System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, HeaderStringValues>>.GetEnumerator() { throw null; }
+        System.Collections.IEnumerator System.Collections.IEnumerable.GetEnumerator() { throw null; }
+        public struct Enumerator : System.Collections.Generic.IEnumerator<System.Collections.Generic.KeyValuePair<string, HeaderStringValues>>
+        {
+            public bool MoveNext() { throw null; }
+            public readonly System.Collections.Generic.KeyValuePair<string, HeaderStringValues> Current { get { throw null; } }
+            public void Dispose() { throw null; }
+            object? System.Collections.IEnumerator.Current { get { throw null; } }
+            void System.Collections.IEnumerator.Reset() { throw null; }
+        }
+    }
+    public readonly struct HeaderStringValues : System.Collections.Generic.IEnumerable<string>
```

Comment:
Is this generated via [ref assembly generator](https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/updating-ref-source.md)?
Is this placed where the generator placed it?

Because if not, it will cause plethora changes once somebody else uses the tool.

Reply:
Oh, I missed it and added it manually. Will generate now. Thanks!